### ISSUE TITLE
Enabling Colour Synching => Object Reference not pointing to Object Instance

### ISSUE THREAD

**Issue by SVNKoch** [phw198/OutlookGoogleCalendarSync#1424]
**OGCS Version**: v2.9.4.0  
**Edition**: Installed 
**Sync Direction**: O->G

**Bug Description**  
Enabling synching of colour throws error. Regardless of presence of mappings or not.
Disabling the colour synching again => works as expected.

Error as Text:

> Unable to update existing entries in the Google calendar.
> Der Objektverweis wurde nicht auf eine Objektinstanz festgelegt.

![grafik](https://user-images.githubusercontent.com/39488412/163249536-3a7e29d3-6120-4c5c-9aaa-1a278579bed8.png)
![grafik](https://user-images.githubusercontent.com/39488412/163249628-ba2d7796-b0a9-49f9-b5de-b64ebd9bb805.png)
[error.log](https://github.com/phw198/OutlookGoogleCalendarSync/files/8483920/error.log)

**Comment by SVNKoch** [comment]
the part in the settings.xml looks like:
```
<CloakEmail>true</CloakEmail>
<ColourMaps />
<ConfirmOnDelete>false</ConfirmOnDelete>
```
just adding it as I manually deleted the mappings because there is no way to do it via the UI (which would be an improvment too 😉 )

**Comment by paulandersonsco** [comment]
I had the same issue with the new Alpha version. I've disabled colours/catrgories for now and I'm not getting the error. Thanks for the heads up on how to work around this SVNKoch.

**Comment by phw198** [comment]
This should be a duplicate of #1422, which now has a hotfix posted.

@SVNKoch Is it not possible to delete the mapping by opening the mapping screen, highlighting each row and pressing delete, then Save? But yes, seems like a sensible enhancement, given someone else wasn't sure how to remove a mapping.

**Comment by phw198** [comment]
[baa65bb](https://github.com/phw198/OutlookGoogleCalendarSync/commit/baa65bbfa4d6c71fd4b7160e8b1b984c27486a1b) released in v2.9.5 has added a "Remove row" button, so I will close this issue.

Also relates to #1389, #1466
